diff --git a/node_modules/prismjs/plugins/autoloader/prism-autoloader.js b/node_modules/prismjs/plugins/autoloader/prism-autoloader.js
index 964e767..5043efc 100644
--- a/node_modules/prismjs/plugins/autoloader/prism-autoloader.js
+++ b/node_modules/prismjs/plugins/autoloader/prism-autoloader.js
@@ -192,21 +192,7 @@
 
 	var script = Prism.util.currentScript();
 	if (script) {
-		var autoloaderFile = /\bplugins\/autoloader\/prism-autoloader\.(?:min\.)js$/i;
-		var prismFile = /[\w-]+\.(?:min\.)js$/i;
-		if (script.hasAttribute('data-autoloader-path')) {
-			// data-autoloader-path is set, so just use it
-			languages_path = script.getAttribute('data-autoloader-path').trim().replace(/\/?$/, '/');
-		} else {
-			var src = script.src;
-			if (autoloaderFile.test(src)) {
-				// the script is the original autoloader script in the usual Prism project structure
-				languages_path = src.replace(autoloaderFile, 'components/');
-			} else if (prismFile.test(src)) {
-				// the script is part of a bundle like a custom prism.js from the download page
-				languages_path = src.replace(prismFile, 'components/');
-			}
-		}
+		var languages_path = '/typo3conf/ext/codehighlight/Resources/Public/Prism/components/';
 	}
 
 	var config = Prism.plugins.autoloader = {
